eval/benchmark_mipnerf360_mcmc.sh

```diff
@@ -40,7 +40,7 @@ do
         --eval \
         --headless \
         --save-eval-images \
-        --strategy mcmc
+        --config .eval/mcmc_optimization_params.json
```

Comment:
The dot must be removed. Should be
        --config eval/mcmc_optimization_params.json
Otherwise it will fail to load it.